README.md

```diff
@@ -29,58 +30,147 @@ of the resulting orbits.
 In OOrb, tools for making ephemeris predictions and classification of
 objects (i.e., NEO-MBO-TNO) are also available.
 
-# Installation using conda #
+# Installation using `conda` #
 
-The easiest way to install OOrb on Linux and OSX 64-bit systems is using the conda installer, which requires some form of this package manager to be installed on your system (e.g., conda, anaconda, miniconda):
+The easiest way to install OOrb on Linux and OSX 64-bit systems is using the
+conda installer, which requires some form of this package manager to be
+installed on your system (e.g., conda, anaconda, miniconda):
 
-> `conda config --add channels conda-forge`
+```bash
+conda install -c defaults -c conda-forge openorb
+```
 
-> `conda install openorb`
+For more details on the OOrb conda package please refer to [this
+website](https://github.com/conda-forge/openorb-feedstock).
 
-For more details on the OOrb conda package please refer to [this website](https://github.com/conda-forge/openorb-feedstock).
+## Installing additional ephemerides files ##
 
-# Manual Installation #
+OpenOrb comes with JPL's DE430 by default. Additional ephemerides 
+can be installed using:
 
-## Requirements ##
+```bash
+conda install openorb-data-de405
```

Comment:
Thanks, good catch! Will fix.

N.b., the reason why I changed the docs to include the more complicated `-c defaults -c conda-forge` segment is because `conda-forge` has a tendency to replace a lot of packages that come with Anaconda with their own versions. This is something a casual user probably doesn't want.